.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,88 @@
+      - main
+  push:
+    branches:
+      - main
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+  workflow_dispatch:
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.daemon=false
+  REGISTRY: ghcr.io
+
+jobs:
+  disk_space:
+    name: Docker Build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Free Disk Space
+        uses: ./.github/actions/free-disk-space
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-home-cache-cleanup: true
+      - name: Get ORT version
+        run: |
+          ORT_VERSION=$(./gradlew -q properties --property version | sed -nr "s/version: (.+)/\1/p")
+          echo "ORT_VERSION=${ORT_VERSION}" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract Docker Metadata for ort Image
+        id: meta-ort
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.REGISTRY }}/${{ github.repository_owner }}/ort
+          tags: |
+            type=pep440,pattern={{version}}
+            type=raw,value=${{ env.ORT_VERSION }}
+            type=ref,event=branch
+            type=ref,event=tag
+            type=sha
+      - name: Build & Push ort Docker Image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta-ort.outputs.tags }}
+          labels: ${{ steps.meta-ort.outputs.labels }}
+          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ github.repository_owner }}/ort:cache
+          cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ github.repository_owner }}/ort:cache,mode=max
+          build-args: ORT_VERSION=${{ env.ORT_VERSION }}
+      - name: Extract Docker Metadata for ort-minimal Image
+        id: meta-ort-minimal
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.REGISTRY }}/${{ github.repository_owner }}/ort-minimal
+          tags: |
+            type=pep440,pattern={{version}}
+            type=raw,value=${{ env.ORT_VERSION }}
+            type=ref,event=branch
+            type=ref,event=tag
+            type=sha
+      - name: Build & Push ort-minimal Docker Image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta-ort-minimal.outputs.tags }}
+          labels: ${{ steps.meta-ort-minimal.outputs.labels }}
+          target: minimal
+          cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ github.repository_owner }}/ort:cache
+          build-args: ORT_VERSION=${{ env.ORT_VERSION }}
+      - name: Print Disk Space
+        run: df -h
```

Comment:
Just double-checking: I guess this is intended here, and not a debug left-over?

Reply:
Yes, I thought it's useful in case we see out of disk space issues again, then we can easily check how much disk space was left at the end of the last successful job. But I have the impression that Docker deletes some temporary data after the build as the last time I checked, the value was higher than expected, so I could also remove it.